README.md

```diff
@@ -1,6 +1,9 @@
 # RHOAS CLI
+<p align="center">
+  <img alt="Logo" src="https://user-images.githubusercontent.com/11743717/127519981-97c76ae4-f17b-4ac8-8b4d-365bfa4a6374.png">
```

Comment:
Is this ID coming from GitHub profile page?

In situations when proxy is used github content might take long time to load. Anyway we can load content locally.

Since logo is done in svg I think the best to put it inline as base64 image+svg